src/main/java/Duke.java

```diff
@@ -1,10 +1,196 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final String DASHES = "____________________________________________________________";
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|";
+    private static final String MESSAGE_WELCOME = " Hello! I'm Duke! Tell me what to do! Available commands: todo, deadline, event, list, bye";
+    public static final String COMMAND_LIST_WORD = "list";
+    public static final String COMMAND_COMPLETED_WORD = "done";
+    public static final String COMMAND_TODO_WORD = "todo";
+    public static final String COMMAND_DEADLINE_WORD = "deadline";
+    public static final String COMMAND_EVENT_WORD = "event";
+    public static final String COMMAND_EXIT_WORD = "bye";
+    public static final String MESSAGE_GOODBYE = " Bye! Do visit next time! More upgrades coming soon :-)";
+    public static final int TASK_DATA_COUNT = 2;
+    public static final int TASK_DATA_INDEX_DESCRIPTION = 0;
+    public static final int TASK_DATA_INDEX_ADDITIONAL_INFO = 1;
```

Comment:
A lot of constants have been declared here. Could possibly split them into different blocks and add some form of javadoc documentation.